### PR TITLE
docs(readme): updated README based on new setup

### PR DESCRIPTION
**Pull request by joonaun93**
Updated the README based on my own setup.

### CODE REVIEW

**Comment by sebastiangaertner** [comment]
@Kpoke could you check this please and merge if correct. For example the non ssl get parameter is for anyone the same or might there be differences depending on your environment and settings?

File: README.md [hunk 1]
```diff
@@ -475,22 +487,51 @@ NOTE: There is another command: `test-watch-debug`, it is the same with `test-wa
...
+
+2. If you run into internal errors code 500 related to JWT/PEM while testing the API, navigate to the server/services/JWTService.js file and replace the following:
+
+```
+const privateKEY = process.env.PRIVATE_KEY;
+const publicKEY = process.env.PUBLIC_KEY;
+
+//to
+
+const privateKEY = process.env.PRIVATE_KEY.replace(/\\n/g, '\n');
```
Comment (Kpoke) [inline README.md:530]: You can replace this in the file itself,  so there won't be a need for this

File: README.md [hunk 2]
```diff
@@ -475,22 +487,51 @@ NOTE: There is another command: `test-watch-debug`, it is the same with `test-wa
 
 Can also use Postman to test the API in a more real environment. Import the API spec from [here](https://github.com/Greenstand/treetracker-wallet-api/blob/master/docs/api/spec/treetracker-wallet-api-v1-10.yaml).
 
-To run a local server with some seed data, run command:
+1. To set up a local server with some seed data, temporarily copy the following (from serverTest.js) and paste into server/server.js:
```
Comment (Kpoke) [inline README.md:490]: can you add a new script and remove the serverTest.js file altogether? So to set up a local server with some seed data would still be npm run server-test but the server-test script would be `npm run test-seed and npm run server`

File: README.md [hunk 3]
```diff
@@ -57,22 +57,34 @@ You can use docker-compose, to start a database. To do that:
 }
 ```
 
-3. Copy .env.example to .env, and update the DATABASE_URL environment variable
+3. Create an .env.development file in the root "/" directory and copy the contents of .env.example. Then update the DATABASE_URL environment variable
 
 ```
-DATABASE_URL=postgresql://wallet_user:secret@localhost:5432/wallet_user?ssl=no-verify
+DATABASE_URL=postgresql://wallet_user:secret@localhost:5432/wallet_user
```
Comment (Kpoke) [inline README.md:63]: Can you make it `?false` or put it as an option?  Also, can you add a disclaimer that ssl=no-verify should be used if the database is not locally hosted

**Review by Kpoke — CHANGES_REQUESTED** [review]

**Comment by joonaun93** [comment]
Hi @Kpoke, made the changes according to the comments.

File: README.md [hunk 4]
```diff
@@ -60,7 +60,9 @@ You can use docker-compose, to start a database. To do that:
 3. Create an .env.development file in the root "/" directory and copy the contents of .env.example. Then update the DATABASE_URL environment variable
 
 ```
-DATABASE_URL=postgresql://wallet_user:secret@localhost:5432/wallet_user
+DATABASE_URL=postgresql://wallet_user:secret@localhost:5432/wallet_user?ssl=false
```
Comment (Kpoke) [inline README.md:63]: it is actually ?false not ?ssl=false

**Pull request merged**